web/src/main/java/se/inera/intyg/webcert/web/csintegration/integration/CSIntegrationService.java

```diff
@@ -187,6 +199,33 @@ public Certificate createCertificate(CreateCertificateRequestDTO request) {
         return response.getCertificate();
     }
 
+    public CreateDraftCertificateResponseType createDraftCertificate(CreateCertificateRequestDTO request, IntygUser user) {
```

Comment:
Maybe I remember wrong but I remember in the design meeting we said we could use the same create endpoint we already have